guides/NIGHTLY_PIPELINES.md

```diff
@@ -0,0 +1,109 @@
+# Nightly Pipelines
+
+This guide outlines how to find, diagnose, take action and report on the RHMI 2.x nightly pipelines. 
+
+It assumes no prior exposure to the integreatly-operator or the pipelines.
+
+
+
+## Resources
+
+- Documentation
+
+  - [Detailed overview of RHMI 2.x nightly pipelines](https://docs.google.com/document/d/1eYtezxJHgOWENs_KRL27gp6AQmggCLUMVkIrSaNIcEQ/edit)
+
+- Pipelines
```

Comment:
There are two more pipelines (but this is subject to change):

- rhmi upgrade -> https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/Nightly/job/rhmi-upgrade
- openshift upgrade (currently disabled due to issues with installing with 4.3.0). This pipeline will likely be changed (there might be two openshift upgrade pipelines - pending discussion on the versions): https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/Nightly/job/rhmi-openshift-upgrade/